docs/configuration.rst

```diff
@@ -4,37 +4,161 @@
+
+    projects:
+      default: 12345
+      prod: 33333
+
+    stack: scrapy:1.3-py3
+
+If you wish to use the stack *only* for project ``12345``, expand its entry in
+``projects`` as follows::
+
+    projects:
+      default:
+        id: 12345
+        stack: scrapy:1.3-py3
+      prod: 33333
+
+The following is a list of all available configuration options:
+
+================ ============================================ =================
+Option           Description                                  Scope
+================ ============================================ =================
+``requirements`` Path to the project's requirements file, and global only
+                 to any additional eggs that should be
+                 deployed to Scrapy Cloud. See
+                 :ref:`deploying-dependencies`.
+``stack``        `Scrapy Cloud stack`_ to use (this is the    global default
+                 environment that your project will run in,   and project-\
+                 e.g. the Scrapy version that will be used).  specific
+``image``        Whether to use a custom Docker image on      global default
+                 deploy. See :ref:`deploy-custom-image`.      and project-\
+                                                              specific
+``version``      Version tag to use when deploying. This can  global only
+                 be an arbitrary string or one of the magic   
+                 keywords ``AUTO`` (default), ``GIT``, or
+                 ``HG``. By default, ``shub`` will
+                 auto-detect your version control system and
+                 use its branch/commit ID as version.
+``apikey``       API key to use for deployments. You will     global only
+                 typically not have to touch this setting as
+                 it will be configured inside
+                 ``~/.scrapinghub.yml`` in your home
+                 directory, via ``shub login``.
+================ ============================================ =================
 
-    # Use git branch/commit as version when deploying. Other possible values
-    # are AUTO (default) or HG
-    version: GIT
-
-``projects`` is a mapping from human-friendly names to project IDs. This allows
-you to run ``shub deploy prod`` instead of ``shub deploy 33333``. ``version``
-is a string to be used as project version information when deploying to Scrapy
-Cloud. By default, ``shub`` will auto-detect your version control system and
-use its branch/commit ID as version.
-
-Typically, you will configure only your API keys and the version specifier in
-the global ``~/.scrapinghub.yml``, and keep project configuration in the local
-``project_dir/scrapinghub.yml``. However, sometimes it can be convenient to
-also configure projects in the global configuration file. This allows you to
-use the human-friendly project identifiers outside of the project directory.
+
+.. _`Scrapy Cloud stack`: http://help.scrapinghub.com/scrapy-cloud/scrapy-cloud-stacks
```

Comment:
It's worth pointing to the new Knowledge base, instead of the old one: https://helpdesk.scrapinghub.com/support/solutions/articles/22000200402-scrapy-cloud-stacks